schema/test/unit/tables/ciip_user_organisation_test.sql

```diff
@@ -126,7 +126,7 @@ select results_eq(
   $$
     select count(*) from ggircs_portal.ciip_user_organisation
   $$,
-  ARRAY[5::bigint],
+  ARRAY[8::bigint],
```

Comment:
Not a change request, just some useful info for future tests: Using ARRAY[...] in these results_eq() assertions was giving me some trouble when the array of results contained different types (not an issue here). I've recently started using values(...) instead of ARRAY[...] which appears to work better.